### ISSUE TITLE
Bug: When logged in, no `h1` element present

### ISSUE THREAD

**Issue by milofultz** [codeforpdx/PASS#519]
### Bug Description:

When logged in, viewing the page with "Contacts"/"Civic Profile" tabs, there is no `h1` for the page.

### Reproduction Steps:    

**1.** Log in.
**2.** On next page when logged in, check the DOM.

### Expected Behavior:

There should be an `h1` that "describes the content of the page (similar to the document's [<title> element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/title))" ([from MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements#avoid_using_multiple_h1_elements_on_one_page)).

### Possible Solution (optional):

Add an `h1` that describes the content of the page. Don't know if it should be for the entire page (including the tabbed areas of "Contacts"/"Civic Profile") or just the tab themselves. Think it should probably be the whole page.

**Comment by xscottxbrownx** [comment]
They are currently two separate pages.

<img width="203" alt="Screenshot 2023-11-21 at 9 37 18 AM" src="https://github.com/codeforpdx/PASS/assets/71395931/305ce31f-7fa6-4302-bc17-b940301e0515">

<img width="756" alt="Screenshot 2023-11-21 at 9 33 41 AM" src="https://github.com/codeforpdx/PASS/assets/71395931/283707ee-1d38-4b67-b69f-0a8d501dd2ec">

**Comment by leekahung** [comment]
> They are currently two separate pages.

That's right. Currently how the site is set up so that "Contacts" is in a separate page with the contacts list and "Civic Profile" will be on their own page with profile information. The two pages can be accessible via the navbar which consist of MUI tabs linking the pages to their respective routes (`/contacts` and `/civic-profile` as Scott pointed out in the image of `src/AppRoutes.jsx`).

So if we were to improve the site with `h1` elements for accessibility, think we'll needed them each of the pages, one for "Contacts" and one for "Civic Profile". As for what elements to utilize, I think the recently implemented breadcrumbs would be a good choice since the trailing crumb would typically display the name of the route they've entered.

<img width="1440" alt="Screen Shot 2023-11-21 at 12 49 30" src="https://github.com/codeforpdx/PASS/assets/14917816/c2285fdf-b934-4141-bd29-dcd23097bed7">

<img width="1440" alt="Screen Shot 2023-11-21 at 12 49 45" src="https://github.com/codeforpdx/PASS/assets/14917816/e3a3d02a-61b3-42e1-a322-4a57d4825afa">

The only part which this might not work would be under contacts since the breadcrumb there utilizes the contact's full webId as the crumb. We'll probably need an alternative for that edge case.

<img width="1440" alt="Screen Shot 2023-11-21 at 12 51 51" src="https://github.com/codeforpdx/PASS/assets/14917816/a9a7e38e-68a7-4861-b0ae-5208a2e84ccc">
